vendor/github.com/golang/protobuf/jsonpb/jsonpb.go

```diff
@@ -818,7 +818,7 @@ func (u *Unmarshaler) unmarshalValue(target reflect.Value, inputValue json.RawMe
 
 			t, err := time.Parse(time.RFC3339Nano, unq)
 			if err != nil {
-				return fmt.Errorf("bad Timestamp: %v (%+v)", err, prop)
+				return fmt.Errorf("bad Timestamp: %v", err)
```

Comment:
was this intentional?

Reply:
I ran a `dev ensure` when I thought I was going to be adding a dependency, but then I ended up going a different direction and removed the dep. Proto was apparently updated with a `dev ensure` so I left it in (this seems to be the only change?) but I'm just as happy to remove it.